commands/timeSinceThatcher.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    name: 'timeSinceThatcher',
+    description: 'prints days since thatcher\'s death in days, and an image of her grave',
+    
+ execute(message, args) {
+let timeOfDeath = Date.parse('2013/04/08');
+let timeNow = Date.now();
+const difference = timeNow -  timeOfDeath;
+let numOfDays = 0;
+
+if (message == "-thatcher") {
```

Comment:
if you swap this to args[0] === "--thatcher", it could potentially feature other figures
also message is a json object which stems from client so it'd have to be message.content iirc

Reply:
changed it to args[0] == "'--thatcher" , updating soon